angular/src/app/account/send-sidechain/address/send-address.component.html

```diff
@@ -50,6 +48,7 @@ <h3>{{ "Account.FederationSwapAddress" | translate }}: {{ sendService.address }}
   </mat-form-field>
 
   <div class="confirmation-label" *ngIf="confirmationMessage">{{ confirmationMessage }}</div>
+  <p><mat-checkbox formControlName="sidechainFee" [(ngModel)]="sendSidechainService.includeFee">Include sidechain fee</mat-checkbox></p>
```

Comment:
Maybe consider having some tooltip that explains what this checkbox does. Not needed now, but perhaps in the future.